### PR TITLE
Closes #2254 - README updates for 1.0

### LINKED ISSUE #2254: Documentation: `README` updates ahead of 1.0
### Please select a category the issue is focused on?

_No response_

### Let us know where something needs a refresh or put your idea here!

- [x]  Update list of authors/contributors
- [x]  Update conferences list
- [x]  Update Release Schedule
- any other updates to the `README`

### PR DESCRIPTION
**Pull request by manciniedoardo**
Closes #2254 

Thank you for your Pull Request! We have developed this task checklist from the [Development Process Guide](https://pharmaverse.github.io/admiraldev/articles/development_process.html) to help with the final steps of the process. Completing the below tasks helps to ensure our reviewers can maximize their time on your code as well as making sure the admiral codebase remains robust and consistent.   

Please check off each taskbox as an acknowledgment that you completed the task or check off that it is not relevant to your Pull Request. This checklist is part of the Github Action workflows and the Pull Request will not be merged into the `devel` branch until you have checked off each task.

- [x] Place Closes #<insert_issue_number> into the beginning of your Pull Request Title (Use Edit button in top-right if you need to update)
- [x] Code is formatted according to the [tidyverse style guide](https://style.tidyverse.org/). Run `styler::style_file()` to style R and Rmd files
- [x] Updated relevant unit tests or have written new unit tests, which should consider realistic data scenarios and edge cases, e.g. empty datasets, errors, boundary cases etc. - See [Unit Test Guide](https://pharmaverse.github.io/admiraldev/articles/unit_test_guidance.html#tests-should-be-robust-to-cover-realistic-data-scenarios)
- [x] If you removed/replaced any function and/or function parameters, did you fully follow the [deprecation guidance](https://pharmaverse.github.io/admiraldev/articles/programming_strategy.html#deprecation)?
- [x] Review the [Cheat Sheet](https://github.com/pharmaverse/admiral/blob/main/inst/cheatsheet/admiral_cheatsheet.pdf). Make any required updates to it by editing the file `inst/cheatsheet/admiral_cheatsheet.pptx` and re-upload a PDF version of it to the same folder.
- [x] Update to all relevant roxygen headers and examples, including keywords and families. Refer to the [categorization of functions](https://pharmaverse.github.io/admiraldev/articles/programming_strategy.html#categorization-of-functions) to tag appropriate keyword/family.
- [x] Run `devtools::document()` so all `.Rd` files in the `man` folder and the `NAMESPACE` file in the project root are updated appropriately
- [x] Address any updates needed for vignettes and/or templates
- [x] Update `NEWS.md` under the header `# admiral (development version)` if the changes pertain to a user-facing function (i.e. it has an `@export` tag) or documentation aimed at users (rather than developers)
- [x] Build admiral site `pkgdown::build_site()` and check that all affected examples are displayed correctly and that all new functions occur on the "[Reference](https://pharmaverse.github.io/admiral/cran-release/reference/index.html)" page. 
- [x] Address or fix all lintr warnings and errors - `lintr::lint_package()`
- [x] Run `R CMD check` locally and address all errors and warnings - `devtools::check()`
- [x] Link the issue in the Development Section on the right hand side.
- [x] Address all merge conflicts and resolve appropriately
- [x] Pat yourself on the back for a job well done! Much love to your accomplishment!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -151,20 +160,20 @@ All `{admiral}` functions are easily findable.
...
-* Please see the [FAQ](https://pharmaverse.github.io/admiral/articles/faq.html) for the most frequent questions
-* Please see the [Contribution Model](https://pharmaverse.github.io/admiral/articles/contribution_model.html) for how to get involved with making contributions
-* Please see [FAQ: R and Package Versions](https://pharmaverse.github.io/admiral/articles/faq.html#why-do-we-use-a-certain-r-version-and-package-versions-for-development) for why we develop with certain R and package versions.
+* Please see the [Programming Strategy](https://pharmaverse.github.io/admiraldev/articles/programming_strategy.html) to understand how functions are created.
+* Please see the [FAQ](https://pharmaverse.github.io/admiral/articles/faq.html) for the most frequent questions.
+* Please see the [Contribution Model](https://pharmaverse.github.io/admiral/articles/contribution_model.html) for how to get involved with making contributions.
+* Please see [FAQ: R and Package Versions](https://pharmaverse.github.io/admiral/articles/faq.html#why-do-we-use-a-certain-r-version-and-package-versions-for-development). for why we develop with certain R and package versions.
```
Comment (bms63) [inline README.md:176]:
```suggestion
* Please see [FAQ: R and Package Versions](https://pharmaverse.github.io/admiral/articles/faq.html#why-do-we-use-a-certain-r-version-and-package-versions-for-development) for why we develop with certain R and package versions.
```

**Review by bms63 — CHANGES_REQUESTED** [review]
couple of minor updates

**Review by bms63 — COMMENTED** [review]
man pages action is failing

**Comment by manciniedoardo** [comment]
> man pages action is failing

Fixed - links now failing but not sure if this is because I added a new page which is currently unpublished

**Comment by bms63** [comment]
> > man pages action is failing
> 
> Fixed - links now failing but not sure if this is because I added a new page which is currently unpublished

For it to become a webpage, we will  need to link it in the pkgdown yaml file so it publishes.

Maybe this should live over in admiraldiscovery instead of on admiral?

**Comment by bms63** [comment]
> > > > man pages action is failing
> > > 
> > > 
> > > Fixed - links now failing but not sure if this is because I added a new page which is currently unpublished
> > 
> > 
> > For it to become a webpage, we will need to link it in the pkgdown yaml file so it publishes.
> > Maybe this should live over in admiraldiscovery instead of on admiral?
> 
> My understanding was that admiraldiscovery would only be for the reactive function discovery table?

We can add whatever we want :)

**Comment by manciniedoardo** [comment]
> > > > > man pages action is failing
> > > > 
> > > > 
> > > > Fixed - links now failing but not sure if this is because I added a new page which is currently unpublished
> > > 
> > > 
> > > For it to become a webpage, we will need to link it in the pkgdown yaml file so it publishes.
> > > Maybe this should live over in admiraldiscovery instead of on admiral?
> > 
> > 
> > My understanding was that admiraldiscovery would only be for the reactive function discovery table?
> 
> We can add whatever we want :)

True, personally I think its place is more in the "Community" tab though.

File: README.md [hunk 2]
```diff
@@ -172,25 +181,26 @@ If you are interested in R and Clinical Reporting, then visit the [pharmaverse b
...
+* [Cross Industry Package Development](https://www.youtube.com/watch?v=M4L1PPMu0pU) (recording from R in Pharma 2023).
+* [Paving the Way for Clinical Submissions in R](https://phuse.s3.eu-central-1.amazonaws.com/Archive/2023/SDE/EU/London/PRE_London09.pdf) (slides from PHUSE SDE in London).
+* [An Overview of {admiral}](https://phuse.s3.eu-central-1.amazonaws.com/Archive/2023/SDE/US/Summit/PRE_Summit03.pdf) (slides from PHUSE SDE in Summit, NJ).
+* [{admiralonco}](https://phuse.s3.eu-central-1.amazonaws.com/Archive/2023/Connect/US/Florida/REC_OS12.mp4) (recording for talk at PHUSE US Connect 2023, slides also available [here](https://phuse.s3.eu-central-1.amazonaws.com/Archive/2023/Connect/US/Florida/PRE_OS12.pdf)).
+* [Programming ADNCA using R and {admiral}](https://phuse.s3.eu-central-1.amazonaws.com/Archive/2023/Connect/US/Florida/REC_OS09.mp4) (recording of presentation from PHUSE US Connect 2023).
+
+For a full collection of `{admiral}` conference presentations over the years, please travel to our [Presentation Archive](presentation_archive.html).
```
Comment (bms63) [inline README.md:194]: i think if you add `articles` to the front of the `presentation_archive.html`  it might work at the publishing time

Comment (bms63) [reply README.md:194]: but i think admiraldiscoverty might still be better

**Comment by bms63** [comment]
Discovering admiral!!  Feels to fit in nicely. but it is just a glorified web page that we link to

**Comment by manciniedoardo** [comment]
> Discovering admiral!! Feels to fit in nicely. but it is just a glorified web page that we link to

Ok I'm swayed 😄 - will move!

**Comment by manciniedoardo** [comment]
I guess the only slightly annoying thing is that then you are having to upkeep two pages for the same material - are you happy to then remove the 2023 conference presentation materials from this section also, and just link directly to admiraldiscovery?

**Comment by bms63** [comment]
> I guess the only slightly annoying thing is that then you are having to upkeep two pages for the same material - are you happy to then remove the 2023 conference presentation materials from this section also, and just link directly to admiraldiscovery?

Yeah!! lets just update in one place!!

**Pull request merged**